conf/routes

```diff
@@ -28,6 +28,7 @@ GET         /feedbackform/send/:sessionid        controllers.FeedbackFormsContro
 GET         /feedbackform/update                 controllers.FeedbackFormsController.update(id: String)
 POST        /feedbackform/update                 controllers.FeedbackFormsController.updateFeedbackForm
 GET         /feedbackform/delete                 controllers.FeedbackFormsController.deleteFeedbackForm(id: String)
+POST        /feedbackform/preview                controllers.FeedbackFormsController.getFeedbackFormPreview
```

Comment:
Getting a feedback form should be GET request and can we use URI as /feedbackform/:id ?